lib/charms/loki_k8s/v0/loki_push_api.py

```diff
@@ -1496,7 +1478,7 @@ def __init__(
         try:
             alert_rules_path = _resolve_dir_against_charm_path(charm, alert_rules_path)
         except InvalidAlertRulePathError as e:
```

Comment:
This looks like a SERIOUS issue. If the path is invalid, we will NEVER write the alerts.